### PR TITLE
Issue 212: change github PR template to adjust both issue# and jira#

### PR DESCRIPTION
**Pull request by jiazhai**
Descriptions of the changes in this PR:
Change PR template to adjust both issue# and jira#

---
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

- [x] Make sure the PR title is formatted like:
    `<Issue #>: Description of pull request`
    `e.g. Issue 123: Description ...`
- [x] Make sure tests pass via `mvn clean apache-rat:check install findbugs:check`.
- [x] Replace `<Issue #>` in the title with the actual Issue number, if there is one.

---

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-xxxxxx
+(PR description content here)...
 
 ---
 Be sure to do all of the following to help us incorporate your contribution
 quickly and easily:
 
 - [ ] Make sure the PR title is formatted like:
-    `<Issue #>: Description of pull request`
+    `<Issue # or JIRA #>: Description of pull request`
```
Comment (eolivelli) [inline .github/PULL_REQUEST_TEMPLATE.md]: Actually we want that prs for jira issues the title starts exaclty with the jira id, BOOKKEEPER-# .
There are some asf gitbot utilities based on this binding so we should report a better example.
IMHO this example is ambiguous for jiras

Comment (jiazhai) [inline .github/PULL_REQUEST_TEMPLATE.md]: Thanks, will do the change.

**Comment by sijie** [comment]
LGTM +1

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (sijie) [inline .github/PULL_REQUEST_TEMPLATE.md]: @eolivelli does this address your comment? do we need to call out 'BOOKKEEPER-#' explicility?

**Comment by eolivelli** [comment]
this closes issue #212

**Pull request merged**